src/main/java/com/comphenix/example/NbtFactory.java

```diff
@@ -50,7 +52,7 @@ private NbtFactory(){
 
                 // Prepare NBT
                 COMPOUND_CLASS = getMethod(0, Modifier.STATIC, offlinePlayer, "getData").getReturnType();
-                BASE_CLASS = COMPOUND_CLASS.getSuperclass();
+                BASE_CLASS = Reflector.getClass(ClassType.NMS, "NBTBase");
```

Comment:
The class name seems to be stable since at least 1.8.8, so chances are high that it will remain that way. There wouldn't be much reason to change it unless they rework the whole NBT system or the maintainers change their mind on acronym capitalisation.

Reply:
So, because you're now looking it up using reflection, does this make it compatible with both 1.9 and 1.13?

Reply:
It now uses a different way to look it up (by name, not by walking from a starting point (OfflinePlayer). The name is stable since at least 1.8.8, so yes.